### ISSUE TITLE
Option to keep toplevel non-collapsible

### ISSUE THREAD

**Issue by WORMSS** [abodelot/jquery.json-viewer#10]
I seen you have the code to make the toplevel collapse..
```javascript
var html = json2html(json, options);
      if (isCollapsable(json))
        html = '<a href class="json-toggle"></a>' + html;
```
I was was thinking of adding `options.toplevelCollapsable != false` This will mean if `toplevelCollapsable` is `true` or absent, it will still allow collapse, if it's set to false, it will not allow it to collapse

**Comment by alessioraggioli** [comment]
Hello, first of all greetings for this library.
I think it could be useful @WORMSS implemention, by the way I have edited the code for my purpose like this:
`if (options.collapsed == true) {
        /* Trigger click to collapse all nodes */
        $(this)
          .find("a.json-toggle:not(':eq(0)')")
          .click();
      }`
Like this you can let the first element non-collapsed, and the others collapsed.
Sorry for bad formatting code, I still don't know how this posts works

**Comment by WORMSS** [comment]
FYI: three backticks ( \` ) on their own line to start a code block, and three again on their own line to end it.

\```
var a = "woohoo";
\```

```
var a = "woohoo";
```

Additionally you can add the language for some syntax highlighting on the first \```

\```javascript
var a = "woohoo";
\```

```javascript
var a = "woohoo";
```

**Comment by alessioraggioli** [comment]
```javascript
if (options.collapsed == true) { /* Trigger click to collapse all nodes */ $(this) .find("a.json-toggle:not(':eq(0)')") .click(); }
```

Thanks ahah 
And how to indent it? :)

**Comment by abodelot** [comment]
Hi,

New option `rootCollapsable` (default: true) has been added. This is included in v1.2.0 release.
